lib/assessment/decision_trees.py

```diff
@@ -367,3 +379,32 @@ def u3l24_modularity_assessment(self, data):
     # or the sprites are not properly created.
     else:
       self.assessment = "No Evidence"
+
+  def u3l28_modularity_functions_assess(self, data):
+    user_functions = [func for func in data["user_functions"] if "draw_loop" not in func]
```

Comment:
Idk if I understand this, iirc draw_loop is a key in the `user_function` dicts, so `"draw_loop"` should always be in `func` right? I would expect something like:
```
[func for func in data["user_functions"] if not func["draw_loop"]]
```